book-store/src/components/books.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    In progress...
+  </div>
+);
```

Comment:
- [ ] As the requirements says you should use  `prop-types` to render the title and the author. not the category

![bbb](https://user-images.githubusercontent.com/61562626/130530749-ccdb25e2-622f-44ff-ba0a-45d5e8832e4d.jpeg)

Reply:
Fixed